packages/web3-validator/src/types.ts

```diff
@@ -63,8 +74,96 @@ export type Web3ValidationOptions = {
 	readonly silent: boolean;
 };
 
-// In `JSONSchemaType` from `ajv` the `type` is required
-// We need to make it optional
-export type JsonSchema = Optional<SomeJSONSchema, 'type'> & {
+// is-my-json-valid types
+export type Json = string | number | boolean | Array<Json> | { [id: string]: Json };
+
+export type Schema = {
```

Comment:
We should document where this came from

Reply:
Just for reference, `is-my-json-valid` has a [types declaration file](https://github.com/mafintosh/is-my-json-valid/blob/master/index.d.ts), but it's not very useful

Reply:
Yep. It is very poor.  I do not want to use it